.gitignore

```diff
@@ -29,7 +29,8 @@ build/
 Pods
 
 # Carthage
-Carthage/Build
+Carthage/
+Cartfile.resolved
```

Comment:
I'm uncertain myself, but thought it probably made sense to ignore it for the same reason you ignore `Gemfile.lock` when writing a rubygem.